### DISCUSSION TITLE
Cypress - Credential Auth

### DISCUSSION THREAD

**Discussion by laurenzhonauer** [nextauthjs/next-auth#3606] [Help]
### Question 💬

In the [Cypress Documentation](https://next-auth.js.org/tutorials/testing-with-cypress) it is stated that if Credential Login is used the `cypress-social-login` dependency is not needed.

It would be nice to also have that case documented there (and/or in this Issue 😄)

In the provided example we see what we can do with the cookie that is returned by the library. How can we achieve this without the library and credential login?

Also how should the data be "stored" / provided to cypress in case JWT is used for authentication?

### How to reproduce ☕️

1. Use https://github.com/vercel/next.js/tree/canary/examples/with-cypress
2. Add next-auth with credential auth
3. Try to create a cypress test that tests authentication with credentials

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by DroopeGu1** [comment]
have you had any feedback or results on this case?
